data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	As the stars wheel outside the mouth of the cave, you note the flow of the ceremony seems to be telling a story of some kind. You understand very few snippets of the words, but the emotion of the singing is clear. The early tone of anticipation shifted to the roar of excitement then on to a peaceful tone of wonder. As the ceremony continues, this shifts to a steady, almost mechanical feel, which is then penetrated by a piercing thread of excitement that leads to the entire tone of the song becoming one of curiosity with a driving beat. The melody wanders through regret and anticipation, mourning and rejoicing, terror and hope, disorder and structure, disappointment and awe, and countless other emotions before finally culminating in a return to the original heartbeat as the singers' voices reach upwards in a tone of hope and then everything trails off into silence.`
+			choice
+				`	(Stand still and absorb the moment.)`
+				`	(Look around to see what is happening next.)`
+			`	Around you, everyone stands still. Most look exhausted, and some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the cave is nearly empty.`
+				goto celebrationend
+			label celebrationannotated
+			`	The flow of people you are a part of joins with other streams of people, finally leading out of the typical hallways and into a large cave. A large opening on one side opens upwards towards the night sky, framed by rough stone and a handful of shield emitters. Stalagmites ring the perimeter of the room with matching stalactites above them, but there is no trace of drips or moisture here. While the walls are rough natural stone, the floor and stalagmites have been polished smooth. At a quick glance, you estimate that there are probably over a hundred people in here with more filtering in through several doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
+			`	Taely gestures quietly to you. "It will start with the ritual query and response asking if we are safe. Safe in this moment, at least."`
+			`	Without warning the room stills, and the doors close. You see a flurry of gestures wash over the room, as if everyone is repeating what someone closer to the center is saying, but you cannot see anyone starting it. "Perimeter secure?" it asks, followed by a ripple of gestures from the edges of the room back toward the center: "All clear."`
+			`	Suddenly you are aware of a drum beat building in the silence, and realize that it had started with the first ripple of gestures, so quiet that it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
+			`	"Thump."`
+			`	"Thump."`
+			`	"Thump."`
+			`	"Thump."`
```

Comment:
I'm cool with the thumping, but I don't quite like the progression idea. There are only a few variations we can do (in fact you listed them: `thump, Thump, THUMP`). Unless there are literally exactly three "thumps" happening, I think the starkness of the word change takes away from the reader's personal ability to interpret "gradualness", if that makes any sense.
Basically, each word change (`thump -> Thump`) would feel more like a leap rather than a slower, more gradual increase to me.

Reply:
Given W1zrad's comments, I prefer to leave this as is. While there is progression here; it is far more than just three thumps. This is more of a crescendo over probably at least a dozen thumps.